src/holofuel/components/layout/PrimaryLayout/PrimaryLayout.js

```diff
@@ -17,9 +17,9 @@ export function PrimaryLayout ({
   children,
   headerProps = {}
 }) {
-  const { data: { holofuelActionableTransactions: transactions = [] } = {} } = useQuery(HolofuelActionableTransactionsQuery)
+  const { data: { holofuelActionableTransactions: transactions = [] } = {} } = useQuery(HolofuelActionableTransactionsQuery, { fetchPolicy: 'network-only' })
```

Comment:
This did the trick?

It's definitely a bummer and we want to keep looking at other ways to do this as we want to get the benefit of caching.

Reply:
@robbiecarlton Honestly, I don't see any other way other than this implementation from @JettTech. 
Because we need to pull this from the network.
Correct me if I am wrong, but the only way this is solved is when we implement `signals` in the Holofuel DNA so that we can use some sort of `subscription` method

Reply:
@robbiecarlton - As we discussed previously, forcing network calls is not ideal, but does work for now.  As we've also discussed, and as @zo-el mentioned here, I do think we can revisit and reform this approach once we implement signals and subscriptions.  I am still open to implementing a more optimal solution, if you know one.